kover-cli/build.gradle.kts

```diff
@@ -21,6 +21,7 @@ plugins {
     id("kover-publishing-conventions")
     id("kover-docs-conventions")
     id("kover-fat-jar-conventions")
+    id("kover-release-conventions")
```

Comment:
Why this plugin have to be applied in sub-projects?

Reply:
Because each subproject has its own directory of `docs` and `examples`, and I want to keep the project isolation